### DISCUSSION TITLE
[Art competition] Buffer Overflow Attack

### DISCUSSION THREAD

**Discussion by froggo-roggo** [prosyslab-classroom/cs348-information-security#402] [Art competition]
![1](https://github.com/prosyslab-classroom/cs348-information-security/assets/82636081/2cdcd2d5-8f64-47b8-8499-010600b76a5d)

## Short Description
For me, the "Buffer Overflow Attack" means "American comic style, a billiard table, colorful billiard balls, a dynamic billiard ball, striking cue ball, white bowling pins, bowling pins, perspective".

## Long Description (up to 3 sentences)
In reality, wouldn't it be absurd if, while playing billiards, suddenly the bowling pins fell down and you scored a strike? Similarly, I wanted to illustrate the process of causing a buffer overflow by precisely manipulating input values, the resulting side effects, and the occurrence of unexpected and absurd bugs that programmers did not intend, in a comic manner.

**Comment by KihongHeo** [comment]
Hi Jisoo,

Nice try. Can you illustrate your idea in a single figure?
This is a hard restriction. Remember that creativity always comes from restriction. 

You don't have to close this issue. You can replace the fig.
